### ISSUE TITLE
Timer Overflows in ulls l0 `copy_submission_events_l0`

### ISSUE THREAD

**Issue by TApplencourt** [intel/compute-benchmarks#1]
Hi,

Thank you very much for this repo. I just tried the `ulls` tests and got a "strange" output:
```
CopySubmissionEvents(api=l0 engine=CCS0)  167089998.336  167088431.200        100.00%        -23.280  334183367.920  [GPU]           [us]
```
It's caused by the timer overflow here:

https://github.com/intel/compute-benchmarks/blob/4426ddcb9a604a81a0d272b53612f07052f591b0/source/benchmarks/ulls_benchmark/implementations/l0/copy_submission_events_l0.cpp#L89

Assuming only one overflow one can use the "adding max value" back trick. For example, 

https://github.com/intel/pti-gpu/blob/fffbe6d142fda7c1776de46cfc8e3d01e5daa34d/tools/ze_tracer/ze_kernel_collector.h#L590

Hope this helps,
Thomas

**Comment by jandres742** [comment]
thanks @TApplencourt ! ok, yes, we are missing to check for the overflow. Let me try to reproduce it internally and work on a fix.

**Comment by jandres742** [comment]
Fixed in c3f55f4255810754e420322a491420d06d7f71e0
